.github/workflows/quality.yml

```diff
@@ -83,7 +83,8 @@ jobs:
 
       - name: Accessibility checking
         if: steps.files.outputs.all_changed_files_count > 0
-        run: npm run a11y:check ${{ steps.files.outputs.all_changed_files }}
+        # run: npm run a11y:check -- -r -f ${{ steps.files.outputs.all_changed_files }}
```

Comment:
Do you want to keep the commented line ?

Reply:
This is still a draft, so not ready to review for now :)